platform/ios/src/MGLMapView.mm

```diff
@@ -392,6 +392,8 @@ - (IBAction)reloadStyle:(__unused id)sender {
 
 - (void)commonInit
 {
+    MGLInitializeRunLoop();
```

Comment:
Since `-commonInit` forces the shared MGLOfflineStorage object to initialize, is it necessary to also initialize it here?